persistence/src/main/scala/com/lightbend/lagom/javadsl/persistence/cassandra/CassandraPersistentEntityRegistry.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package com.lightbend.lagom.javadsl.persistence.cassandra
+
+import java.util.{ Optional, UUID }
+
+import akka.NotUsed
+import akka.japi.Pair
+import akka.stream.javadsl
+import com.lightbend.lagom.javadsl.persistence.{ AggregateEvent, AggregateEventTag, PersistentEntityRegistry }
+
+/**
+ * Functionality specific to the Cassandra persistent entity registry.
+ */
+trait CassandraPersistentEntityRegistry extends PersistentEntityRegistry {
+
+  /**
+   * A stream of the persistent events that have the given `aggregateTag`, e.g.
+   * all persistent events of all `Order` entities.
+   */
+  def uuidEventStream[Event <: AggregateEvent[Event]](
+    fromOffset:   Optional[UUID],
```

Comment:
A crazy idea is to only have UUID and have conversion utility between UUID and Long, probably by introducing a new `Offset` type. Probably not very intuitive, but wanted to mention that it would be possible.